wallet/cardano-sl-wallet.cabal

```diff
@@ -300,3 +300,27 @@ test-suite cardano-wallet-test
 
   build-tools: cpphs >= 1.19
   ghc-options: -pgmP cpphs -optP --cpp
+
+Test-suite unit-tests
+  type:               exitcode-stdio-1.0
+  default-language:   Haskell2010
+  default-extensions: NoImplicitPrelude
+  hs-source-dirs:     test
+  main-is:            Test.hs
+  other-modules:      Migrations
+  ghc-options:        -Wall -threaded -pgmP cpphs -optP --cpp
+  buildable:          False
+  if os(linux)
+    ghc-options:       -optl-fuse-ld=gold
+    ld-options:        -fuse-ld=gold
+  build-depends:      base
+                    , cardano-sl
+                    , cardano-sl-core
+                    , cardano-sl-txp
+                    , cardano-sl-wallet
+                    , safecopy
+                    , tasty
```

Comment:
@nc6 Do you think there would be a value into not pulling yet another dependency with `tasty` here but simply write another spec using the `hspec-autodiscover` functionality? I'm a bit fan of `tasty` as well, but considering all our woes recently with the dependency number, maybe it's worth considering 🙈